src/couch_peruser/src/couch_peruser.erl

```diff
@@ -222,13 +359,13 @@ handle_info(restart_config_listener, State) ->
 handle_info(_Msg, State) ->
     {noreply, State}.
 
+-spec subscribe_for_changes() -> ok.
```

Comment:
Minor nit: maybe add this after gen_server callbacks or before. Just to keep them together visually.